final_prep/annotations.rb

```diff
@@ -4,40 +4,70 @@
 
 # Build a Bear
```

Comment:
VERY nice annotations here Sam. The precise use of technical vocabulary and attend to detail - 💯